preview-src/ui-model.yml

```diff
@@ -8,19 +8,101 @@ site:
   homeUrl: &home_url http://localhost:5252
   components:
   - &component
-    name: datastax
-    title: PREVIEW Datastax Docs
-    url: '#'
+    name: xyz
+    title: &component_title Preview Docs
+    url: /xyz/6.0/index.html
     versions:
-    - &latest_version
-      url: *home_url
-      version: 'latest'
-      displayVersion: 'latest'
-    latestVersion: *latest_version
+    - &latest_version_xyz
```

Comment:
Most changes to this file are to show the version switcher, copied from the default Antora UI